tests/sample_test/src/main.c

```diff
@@ -0,0 +1,22 @@
+#include <example_lib.h>
+
+#include <zephyr/ztest.h>
+
+ZTEST_SUITE(sample_test_suite, NULL, NULL, NULL, NULL, NULL);
+
+ZTEST(sample_test_suite, test_example_1)
+{
+	zassert_equal(1, 1, "1 should be equal to 1");
+}
+
+ZTEST(sample_test_suite, test_example_2)
+{
+	zassert_not_equal(1, 0, "1 should be not equal to 0");
+}
+
+ZTEST(sample_test_suite, test_example_lib)
+{
+	int result = example_lib_calculate(true, 1, 1);
+
+	zassert_equal(2, result, "Result should be equal to 1, but is %d", result);
+}
```

Comment:
Maybe add a comment here that you are INTENTIONALLY not testing `example_lib_calculate(false, x, y);` so that you can demonstrate code coverage.

Reply:
Good idea!